src/management/projects.ts

```diff
@@ -264,11 +264,22 @@ export async function createCloudProject(
     });
     return projectInfo;
   } catch (err) {
-    logger.debug(err.message);
-    throw new FirebaseError(
-      "Failed to create Google Cloud project. See firebase-debug.log for more info.",
-      { exit: 2, original: err }
-    );
+    if (err.status === 409) {
+      throw new FirebaseError(
+        `Failed to create Google Cloud project because there is already a project with ID ${clc.bold(
```

Comment:
QQ - What command is this error from?  It's from `firebase projects:create`?

Reply:
firebase init
then opt to create a new project
Fails with cryptic 409 instead of telling me I already have a project with that name. Hence the change :-)

Reply:
@rachelsaunders I fixed this because it appears in `firebase init` when you decide to create a new project although I believe this would show up in `projects:create` as well.

Reply:
ugh... I thought this was gonna be an issue when we landed on having folks in the CLI be asked to create a unique Project ID rather than just creating an editable one for them from their Project Name :-/

This is immensely better error messaging, though :-D

Reply:
Reopening...

If this error is coming off the `firebase init` command it might be confusing to say "Google Cloud project" since the command is asking about creating a Firebase project.

In the `firebase projects:create` command... that's specifically to create a fresh Google Cloud project with no Firebase services, so the user is def expecting to see the term "Google Cloud project".

Any way that we can be sure where this error messaging displays?

Reply:
It displays in both commands, changed to more neutral message.